extensions/sql-database-projects/src/common/utils.ts

```diff
@@ -578,3 +578,12 @@ export async function getFoldersInFolder(folderPath: string, ignoreBinObj?: bool
 		return await glob(folderFilter, { onlyDirectories: true });
 	}
 }
+
+/**
+ * Determines if provided value is well-known by the extension and therefore is allowed to be sent in telemetry
+ * @param value Value to check if it's well-known
+ * @returns True if value is a well-known value, false otherwise
+ */
+export function isWellKnownValue(value: string): boolean {
+	return constants.WellKnownValues.includes(value);
```

Comment:
This should probably be case-insensitive comparison.

Reply:
I'm changing this function to do a case-insensitive comparison and return the well-known value. It will return undefined if provided value is not well known. This way we'll only be dealing with the value as defined in `constants.WellKnownDatabaseSources`.